elector/connector.go

```diff
@@ -0,0 +1,14 @@
+package elector
+
+import (
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+// Connector specifies a way to connect to ZK
+type Connector interface {
+	// Connect returns a ZK connection and events channel
+	Connect() (*zk.Conn, <-chan zk.Event, error)
+
+	// Close should ensure the ZK connection is closed.
+	Close() error
```

Comment:
why does this return `error`? asking because it (the error return arg) never appears to be set by the implementations provided by this lib. trying to conform to `io.Closer`?

Reply:
@jdef yes, mostly that, and b/c i thought should anything change about the underlying implementation that would require an error here, i'd hate to change the API afterwards.